packages/dds/tree/src/feature-libraries/chunked-forest/codec/chunkDecoding.ts

```diff
@@ -38,15 +38,22 @@ import {
 	type EncodedValueShape,
 	SpecialField,
 } from "./format.js";
-import type { IIdCompressor, SessionSpaceCompressedId } from "@fluidframework/id-compressor";
+import type { IIdCompressor, OpSpaceCompressedId, SessionId } from "@fluidframework/id-compressor";
 
+export interface IdCompressorContext {
```

Comment:
I'd rename this to IdDecodingContext, and I recommend documenting the orginator field, something like "the creator of any local IDs to be decoded"

Reply:
Renamed interface to `IdDecodingContext` and added documentation for originatorId field